package.json

```diff
@@ -139,6 +139,14 @@
 					"dark": "resources/dark/delete.svg"
 				}
 			},
+			{
```

Comment:
I'm actually not sure now, what do you think? This is a non-blocker. so If you are like meh, just ignore me :)

Reply:
I agree that would be better (and did try to make it so).
I tried reversing the order of declaration, but that didn't change anything.

I can't find anything on ordering context menu items. I suspect they are ordered alphabetically, similar to what the groups seem to be (`grp1`, `grp2`, `grp3`).
Also, Intellisense does recognise the schema for the package.json, but doesn't have any suggestions, that would indicate controlling item order.

I suppose I could create another group to force the order, but that would cause another divider.
An even hackier alternative would be to rename the command so it would order the way we want (e.g. prepend `z`: `serviceBusExplorer.zpurgeSubscriptionDeadLetterQueue`)

Lastly, we could leave it the way it is, and I would post an SO question and probably a followup issue.

Reply:
Fair enough!